sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -277,4 +291,23 @@ private static boolean isNdkAvailable(final @NotNull IBuildInfoProvider buildInf
     }
     return null;
   }
+
+  private static boolean isIntegrationAvailable(
+      final @NotNull String integrationClassName,
+      final @NotNull SentryOptions options,
+      final @NotNull LoadClass loadClass) {
+    boolean isAvailable;
+    try {
+      loadClass.loadClass(integrationClassName);
```

Comment:
What if we refactor the `loadClass`, that catches and logs all exceptions and return the boolean only, this block appears duplicated across the code base.

Reply:
sounds good 👍